beacon-chain/rpc/beacon/attestations.go

```diff
@@ -284,7 +284,11 @@ func (bs *Server) StreamIndexedAttestations(
 				if att.Data.Slot < startSlot || att.Data.Slot > endSlot {
 					continue
 				}
-				committee := committeesBySlot[att.Data.Slot].Committees[att.Data.CommitteeIndex]
+				committeesForSlot := committeesBySlot[att.Data.Slot]
```

Comment:
If this is a map you can check `committeesForSlot, ok := committeesBySlot(att.DataSlot]`